src/IK/include/BiomechanicalAnalysis/IK/InverseKinematics.h

```diff
@@ -42,37 +42,32 @@ class HumanIK
      * @param handler pointer to the parameters handler
      * @return true if the SO3 task is initialized correctly
      */
-    bool initializeOrientationTask(
-        const std::string& taskName,
-        const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
+    bool initializeOrientationTask(const std::string& taskName,
+                                   const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
 
     /**
      * initialize the gravity task
      * @param taskName name of the task
      * @param handler pointer to the parameters handler
      * @return true if the gravity task is initialized correctly
      */
-    bool initializeGravityTask(
-        const std::string& taskName,
-        const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
+    bool initializeGravityTask(const std::string& taskName,
+                               const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
 
     /**
      * initialize the R3 task
      * @param taskName name of the task
      * @param handler pointer to the parameters handler
      * @return true if the R3 task is initialized correctly
      */
-    bool initializeFloorContactTask(
-        const std::string& taskName,
-        const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
+    bool initializeFloorContactTask(const std::string& taskName,
+                                    const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
 
-    bool initializeJointRegularizationTask(
-        const std::string& taskName,
-        const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
+    bool initializeJointRegularizationTask(const std::string& taskName,
+                                           const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
 
-    bool initializeJointConstraintsTask(
-        const std::string& taskName,
-        const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
+    bool initializeJointConstraintsTask(const std::string& taskName,
+                                        const std::shared_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> taskHandler);
```

Comment:
I would add the documentation for these

Reply:
Done with commit https://github.com/ami-iit/biomechanical-analysis-framework/pull/31/commits/2a4606a02a2dcf6ed2fa2564db8d7b83456d2c40